script.js

```diff
@@ -0,0 +1,17 @@
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.nav-menu');
+const logo = document.querySelector('.logo');
+const navItem = document.querySelector('.nav-item');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+  logo.classList.toggle('active');
+  navItem.classList.toggle('active');
+});
+
+document.querySelectorAll('.nav-item').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+  logo.classList.remove('active');
+}));
```

Comment:
- [ ] Great job so far, Kindly make sure to add margin to the top and on the right side as shown below.
- [ ] Kindly make sure that the line after all menu options is of the same width as per the Figma design.
- [ ] Kindly make sure to add space after all menu options and before the line as shown.
- [ ] Optional: Kindly follow the same colour for the close button as per the design.

https://user-images.githubusercontent.com/70269507/173751663-a4e17f6a-99e7-41b6-ad4c-59a881e31ff9.mov

Reply:
Updated styling. Matched template.